Add a new ui for setting faction relations.

ChrisPJohn opened the pull request:
![20211109224020_1](https://user-images.githubusercontent.com/7753322/141063491-0d6799ee-8732-4701-8215-821572d6866d.jpg)
![20211109224306_1](https://user-images.githubusercontent.com/7753322/141063504-9257cd61-16cd-47bf-8891-25861e29a3a5.jpg)

ThyWoof commented:
. Instead of UI.AutoWidth() I suggest UI.Width(number) on sliders to move them closer to title strings.
. Consider making the red message italic and a bit smaller to avoid UI horizontal scrolling on default UI mod window sizes.

ChrisPJohn commented:
> . Instead of UI.AutoWidth() I suggest UI.Width(number) on sliders to move them closer to title strings. . Consider making the red message italic and a bit smaller to avoid UI horizontal scrolling on default UI mod window sizes.

I changed it so moving the sliders now triggers all the notifications and achievements (can get spammy with the faction change notifications if you drag them all over the place). So I removed the red message.

As for setting the width on the sliders it had no effect at all (UI.Width(number) had the same with as UI.AutoWidth(), no matter what number I specified).  I moved the names closer to the sliders by adding some padding.

ChrisPJohn commented:
![20211110214900_1](https://user-images.githubusercontent.com/7753322/141245209-5200a08b-9e55-4de7-9ec8-2b1caabd25b4.jpg)

ThyWoof commented on SolastaCommunityExpansion/Viewers/Displays/ToolsDisplay.cs:
```diff
@@ -49,6 +49,44 @@ internal static void DisplayTools()
+                        title = title.white();
+                    }
+                    int intValue = service.FactionRelations[faction.Name];
+                    if (UI.Slider("                              " + title, ref intValue, faction.MinRelationCap, faction.MaxRelationCap, 0, "", UI.AutoWidth()))
```
you could also use UI.Space(number) on cases like this but it would also require an UI.HorizontalScope context. keep in mind for more complex layouts
